lib/rest-router.js

```diff
@@ -0,0 +1,36 @@
+module.exports = RestRouter;
+
+var Trie = require('./trie');
+
+function RestRouter(opts) {
+  this.Trie = new Trie();
```

Comment:
Variable and property names should start with a lower-case letter, i.e. `this.trie`.